tests/src/main/java/com/hazelcast/stabilizer/tests/webContainer/WebClient.java

```diff
@@ -0,0 +1,153 @@
+package com.hazelcast.stabilizer.tests.webContainer;
+
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.annotations.Warmup;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+
+import com.hazelcast.stabilizer.worker.selector.OperationSelector;
+import com.hazelcast.stabilizer.worker.selector.OperationSelectorBuilder;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.CookieStore;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Random;
+
+import static org.junit.Assert.assertEquals;
+
+
+/*
+*
+* A Load producing WebClient test.  making http Put and Get requests,  the the serverIp address / port configured in the
+* properties.  each thread of this test represents a http client / "browser".
+*
+* */
+public class WebClient {
+
+    private final static ILogger log = Logger.getLogger(WebClient.class);
+
+    public String serverIp="";
+    public int serverPort=0;
+
+    public String id;
+    public int threadCount = 3;
+    public int maxKeys=1000;
+    public double getRequestProb = 0.5;
+    public double postRequestProb = 0.5;
+
+    private TestContext testContext;
+
+    private enum RequestType {
+        GET_REQUEST,
+        PUT_REQUEST
+    }
+
+    @Setup
+    public void setup(TestContext testContext) throws Exception {
+        this.testContext = testContext;
+        id = testContext.getTestId();
+
+    }
+
+    @Warmup(global = true)
+    public void warmup() throws Exception { }
+
+    @Run
```

Comment:
Can you make use of the new worker functionality. See IntIntMapTest for a good example.